debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/converters/sink/SinkConverter.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.converters.sink;
+
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.bson.BsonDocument;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.connector.mongodb.converters.sink.LazyBsonDocument.Type;
+import io.debezium.table.ColumnNamingStrategy;
+
+public class SinkConverter {
```

Comment:
Can we use a different name? The difference between `SinkConverter` abs `SinkRecordConverter` is not clear from naming. Esp if this class also operates on `SinkRecord`.

Reply:
`SinkConverter` became `SinkRecordConverter` as it is responsible to determine the correct `SinkValueConverter` based on the Kafka Connect SinkRecord/record.
`SinkRecordConverter` becomes `SinkValueConverter` interface that defines how values get converted into the MongoDB BsonDocument representing the original value.
done.

Reply:
OK